platform/mellanox/mft/templates/control.j2

```diff
@@ -0,0 +1,9 @@
+Package: kernel-mft-dkms-modules-{{kversion}}
+Source: kernel-mft-dkms-dkms-bin
+Version: {{mft_version}}
+Architecture: amd64
+Maintainer: Vivek Reddy <vkarri@nvidia.com>
+Provides: kernel-mft-dkms-modules
```

Comment:
Can you add `Depends: linux-iimage-{{ kversion }}-unsigned` here, so that there's a hard check on the kernel package being installed here?

Reply:
Added